src/ResXManager.VSIX.Compatibility/MoveToResourceConfiguration.cs

```diff
@@ -50,7 +50,8 @@ public class MoveToResourceConfiguration : ItemTrackingCollectionHost<MoveToReso
 {
     public const string Default = @"{""Items"":
 [{""Extensions"":"".cs,.vb"",""Patterns"":""$Namespace.$File.$Key|$File.$Key|StringResourceKey.$Key|$Namespace.StringResourceKey.$Key|nameof($File.$Key), ResourceType = typeof($File)|ErrorMessageResourceType = typeof($File), ErrorMessageResourceName = nameof($File.$Key)""}
-,{""Extensions"":"".cshtml,.vbhtml,.razor"",""Patterns"":""@$Namespace.$File.$Key|@$File.$Key|@StringResourceKey.$Key|@$Namespace.StringResourceKey.$Key""}
+,{""Extensions"":"".cshtml,.vbhtml"",""Patterns"":""@$Namespace.$File.$Key|@$File.$Key|@StringResourceKey.$Key|@$Namespace.StringResourceKey.$Key""}
+,{""Extensions"":"".razor"",""Patterns"":""@localizer[\""$Text\""]""}
```

Comment:
Shouldn't .razor files still support the robust, early binding way with generated code, and not only the error prone @localizer approach.

Reply:
Thank you.

I just pushed another commit to support ResXFileCodeGenerator generated files in razor files.